vendor-extra/LiberoPageBundle/src/Resources/translations/messages+intl-icu.en.xlf

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 http://docs.oasis-open.org/xliff/v1.2/os/xliff-core-1.2-strict.xsd">
+
+    <file source-language="en" datatype="plaintext" original="file.ext">
+
+        <body>
+
+            <trans-unit id="1" resname="libero.page.site_name">
+                <source>Site Name</source>
+            </trans-unit>
+
+            <trans-unit id="2" resname="libero.page.page_title">
```

Comment:
Maybe should be a more specific name? It's for the `<title>` in the HTML `<head>`, so maybe `libero.page.head.title`?